src/components/Carousel/index.js

```diff
@@ -17,7 +17,7 @@ const defaultSettings = {
 
 export default class Carousel extends Component {
   static propTypes = {
-    slidesDataSource: PropTypes.arrayOf(PropTypes.shape({})),
+    data: PropTypes.arrayOf(PropTypes.shape({})),
```

Comment:
Is this intencional or was overwritten by the merge?

Reply:
Intentional, to fix the carousel.
Isn't working now, you're using the prop `data` in the story:

<img width="1037" alt="screen shot 2018-07-02 at 10 52 34 am" src="https://user-images.githubusercontent.com/18464392/42168143-87107284-7de6-11e8-8688-e4773599b659.png">

But instead of changing the prop name in the component we can change the story to inject the prop `slidesDataSource`. What do you prefer?

Reply:
Oops. I think the change in the story will do better

Reply:
Fixed